CMakeModules/leap-config.cmake.in

```diff
@@ -1,3 +1,4 @@
 list(APPEND CMAKE_MODULE_PATH @EOS_ROOT_DIR@/cmake/leap)
 include(EosioTester)
+include(${CMAKE_CURRENT_LIST_DIR}/ship-targets.cmake)
```

Comment:
Does this mean if a user just wants to use the ship targets, LLVM & Boost are still required? `find_dependency()` seems to suggest
> `find_dependency` forwards the correct parameters for `QUIET` and `REQUIRED` which were passed to the original `find_package()` call

Since the original calls were `REQUIRED` it seems like they'd be `REQUIRED` here too? Is there any reasonable way to allow a user to consume the ship target w/o llvm, boost, etc?

Reply:
It is possible to do so using COMPONENTS so when user uses `find_package(leap REQUIRED COMPONENT ship-protocol)` would be consume ship target without llvm, boost. However, this approach doesn't solve the installation dependency. The other way is to treat ship as a different package so that we could create something like `ship-protocol.deb` without llvm and boost dependency.

Reply:
That's true, if a user is utilizing this mechanism they either are 1) using a builddir which implies they built leap ergo have boost & llvm, or 2) using leap-dev.deb which already indicates llvm & boost is a dep so they have that too. So really a non-issue.